data/variables.yaml

```diff
@@ -1,5 +1,5 @@
 tenant_id: default
-server_id: openbanking
+server_id: cdr
```

Comment:
this change should not be commited

Reply:
I have reverted it. However, this is dynamically updated at runtime, so I am not sure the value of changing or setting it beforehand. Perhaps I have misunderstood something? Could you clarify what the reason for preserving a value in the repo is? Thanks